chalicelib/checks/badge_checks.py

```diff
@@ -203,6 +207,11 @@ def yellow_flag_biosamples(connection, **kwargs):
                         messages.append('Biosample is a stem cell line over 10 passages but missing karyotype')
                     elif not passages:
                         messages.append('Biosample is a stem cell line with unknown passage number missing karyotype')
+                if tier and bcc.get('culture_start_date', '2000-01-01') > fbs_chk_date:
+                    valid_fbs = ["VWR 97068-091 Lot 035B15 (phase 1)", "Peak Serum PS-FBS2 Lot 21E1202 (phase 2)", "VWR 89510-184 lot 310B19 (phase 2)"]
+                    fbs_info = bcc.get('fbs_vendor_lot', '').strip()
+                    if fbs_info not in valid_fbs:
+                        messages.append('Tiered cell line cultured after {} missing required FBS vendor and lot info'.format(fbs_chk_date))
```

Comment:
I believe the way that these checks deal with the handling of the messages uses the 'missing ' string and anything that follows in the brief output so how about if I change 'required' to '4DN specified'?

Reply:
I think what you are referring to is in Line 192, so only valid for some cases. In general, it looks like the entire content of `messages` variable ends up in the badge (see e.g. https://data.4dnucleome.org/biosamples/4DNBSIOKOT8F/#badges). Anyway, your proposed edit sounds good to me.

Reply:
Actually I was referring to line 249.  When I was testing with my original version of the message that was similar to the one you proposed but completely lacked the word 'missing' a valueError exception was thrown.  It appears that while not ideal the 'missing' string is required and the text after that is used in the brief output - and yes you are correct that the entire message does become part of the badge so it's just the brief output generation that is affected.  Maybe it would be better to change this dependent behavior but as I've only finally got back to looking at this 3 hours after you replied I think for now I'll go with just updating the message as I proposed.